ihatemoney/forms.py

```diff
@@ -216,6 +218,15 @@ def save(self):
             logging_preference=self.logging_preference,
             default_currency=self.default_currency.data,
         )
+        # Check that the member name is not empty so you don't insert an empty
+        # Person object if no member name is given
+        if self.member_name.data != "":
```

Comment:
I think that this is Falsey, so we could use `if self.member_name.data:` rather than checking for an empty string.